include/chemfiles/exports.h

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
This file should not be committed, it is generated by cmake as needed.

Reply:
Same here, a mistake I am removing these two files from here in my next commit.